src/kaleidoscope/plugin/LEDEffect-BootAnimation.cpp

```diff
@@ -60,7 +60,7 @@ EventHandlerResult BootAnimationEffect::afterEachCycle() {
       Key k = Layer.lookupOnActiveLayer(r, c);
       Key g;
       g.flags = 0;
-      g.keyCode = pgm_read_word(&greeting_[current_index_]);
+      g.keyCode = pgm_read_byte(&greeting_[current_index_]);
```

Comment:
This was an actual error. Reading a word and assigning it to a byte does not make sense.

Reply:
Maybe lift this to its own tiny PR we can merge in now?